js/Learn JavaScript concepts by building a random quote generator.md

```diff
@@ -0,0 +1,289 @@
+Learning JavaScript is fun. But What's the point if you are not building stuff with it.
+So in this article, we will learn javascript basic concepts and create a random quote generator of our own.
+
+I assume you have a basic knowledge of HTML and CSS  as the main objective of this article is to learn javascript. Else you can check these tutorials on [HTML](https://youtu.be/UB1O30fR-EE) and [CSS](https://youtu.be/yfoY53QXEnI). I will try to explain everything in detail and by the time you finish this lesson, you will have an understanding of concepts like strings, arrays, functions, event listeners, template literals in JavaScript. So let's begin.
+
+
+## Getting started
+Lets first create three different files in our folder as 
+- index.html for creating the UI and displaying the content,
+- style.css to make it attractive by adding styles.
+- script.js will contain the actual code which makes our app work.
+
+
+## 1. creating the content of the app
+The first step is to create a UI. Here is our index.html file look like 
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" href="icons/favicon.png">
+    <title>Random quote</title>
+</head>
+<body>
+    <div class="content">
+        <div class="card">
+          <h2>Random quote</h2>
+          <p>wanna see a random quote ?</p>
+          <!--button to display random quote-->
+          <button class="btn">Generate</button>
+
+        </div>
+        <div id="quote">
+         <!--Quote will be displayed here.-->
+        </div>
+    </div>
+</body>
+<script src="script.js"></script>
+</html>
+```
+
+![Screenshot (238).png](https://cdn.hashnode.com/res/hashnode/image/upload/v1585025931498/AIvbN-aiu.png)
+
+And this is how it looks like in the browser. Wait a minute, It doesn't look nice right. 
+## 2. Styling the content
+Let's make it appealing by adding our style.css file
+
+```css
+body{background-color: indigo;  margin:auto;}
+h2{background-color: #1c1c1c; color:#ffffff;
+padding:10px;
+font-family: arial;
+text-align: center;}
+
+.btn{height:50px; width:150px;
+font-size: 18px;
+color:#ffffff;
+background-color: dodgerblue;
+border:none;
+padding:5px;
+border-radius: 5px;}
+
+.card{padding:15px 30px 15px 30px;
+margin:40px auto;
+max-width:500px;
+max-height:300px;
+text-align: center;
+background-color: #ffffff;
+border-radius: 5px;
+} 
+
+p{font-family:'Segoe UI';
+font-weight:400;
+font-size: 20px;}
+
+img{height:25px;
+width:25px;
+}
```

Comment:
Can you just format this CSS file so it becomes more readable, like parenthesis, line-breaks and stuff?